### PR TITLE
Create basic folders

### PR DESCRIPTION
**Pull request by phfernandesmc**
In this branch I created the basic folders that were missing and an action to get the city weather

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 import { Inter } from "next/font/google";
+import "@/style/global.scss"
 
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
-  title: "Weather App",
-  description: "Weather application",
+  title: "Create Next App",
+  description: "Generated by create next app",
```
Comment (Saeka03) [inline src/app/layout.tsx]: Shingo changed the title and description here.
Could you please change it again?
Or if there is something you want, you can change it to whatever you want!

------------
  title: "Weather App",
  description: "Weather application",

**Review by Saeka03 — COMMENTED** [review]
Thanks for that!
I left only one comment, I would like you to check it.

**Review by Saeka03 — APPROVED** [review]
Thanks for modifying it!

**Review by Shingo023 — APPROVED** [review]
Thank you fro creating those folders!

**Pull request merged**